docs/modules/qbi-helper/intro-to-qbi.md

```diff
@@ -0,0 +1,49 @@
+<%(TOC)%>
+
+# Introduction to Query-based interest
+
+<%(Callout message="
+Before reading this document, make sure you are familiar with:
+
+  * [Query-based interest](https://docs.improbable.io/reference/latest/shared/reference/query-based-interest)
+  * [Workers in the GDK]({{urlRoot}}/reference/concepts/worker)
+")%>
+
+This page provides a quick overview of the key primitives in Query-based interest.
+
+## Interest
+
+Query-based interest is enabled for an entity by adding the [`improbable.Interest`](https://docs.improbable.io/reference/latest/shared/schema/standard-schema-library#interest-optional) component to that entity.
+
+`Interest` is effectively a mapping from a component ID to a list of queries, where each list of queries is only active on the worker that is authoritative over the component with that ID.
+
+For example, if you want to control the interest of a worker responsible for simulating the position of a player entity, you could use the `Position` component, whose ID is `54`, as the key to a set of queries. These queries would then define which components the worker should be interested in.
+
+> The [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template) class provides methods to define and manipulate an `Interest` component.
+
+## Queries
+
+A query is represented as a constraint and which components the query should return from entities that satisfy the constraint. You can specify exactly what components to return or state that the query returns _all_ components on matching entities.
+
+> The [`InterestQuery`]({{urlRoot}}/api/query-based-interest/interest-query) class provides methods to create a query, set a constraint and define what components the query should return.
+
+## Constraints
+
+Constraints are used to define what entities a query should be looking for.
+
+The available constraints with Query-based interest are:
+
+|Constraint|Description|
```

Comment:
are these constraints not listed in the normal qbi docs?

Reply:
kind of, but not in a nice way - you have to infer the constraints from the schema provided

https://docs.improbable.io/reference/13.6/shared/reference/query-based-interest#relative-constraints